setup.py

```diff
@@ -70,7 +70,7 @@ def build(cls):
         try:
             import sass
         except ModuleNotFoundError:
-            pip.main(['install', 'libsass'])
+            check_call([sys.executable, '-m', 'pip', 'install', 'libsass'])
```

Comment:
@beraldoleal the `libsass` requirement exists only for building the project package and/or installing the project (from source). Thus it does not make sense to add it to the `requirements.txt` file (since it contains the requirements to run kytos).

That is why this "hardcoded" call is made here, and the `install requirements.txt` is done at some point below on the same file (line 99 at this moment).